constraintlayout/core/src/main/java/androidx/constraintlayout/core/dsl/Chain.java

```diff
@@ -29,11 +29,12 @@ public enum Style {
 
     private Style mStyle = null;
     protected ArrayList<Ref> references = new ArrayList<>();
-    protected Map<Style, String> styleMap = new HashMap<Style, String>() {{
-        put(Style.SPREAD, "'spread'");
-        put(Style.SPREAD_INSIDE, "'spread_inside'");
-        put(Style.PACKED, "'packed'");
-    }};
+    static protected Map<Style, String> styleMap = new HashMap<>();
```

Comment:
`final static protected`

Otherwise, an instance of `Chain` may do something like `styleMap = null`